source/features/clean-header-search-field.tsx

```diff
@@ -0,0 +1,18 @@
+import onetime from 'onetime';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+async function init(): Promise<void> {
+	(await elementReady<HTMLInputElement>('.header-search-input'))!.value = '';
+}
+
+void features.add(__filebasename, {
+	include: [
+		pageDetect.isConversationList,
+		pageDetect.isGlobalConversationList
+	],
+	awaitDomReady: false,
+	init: onetime(init)
```

Comment:
I did not blame this. But what is the reason for the one time?

Reply:
My guess is that the header is outside the ajaxed area, so it doesn't change after navigation. I just tested it and it works:

1. Visit https://github.com/sindresorhus/refined-github/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug
2. Click "795 Closed"
3. Go back

Field still empty